D1673/P1673.md

```diff
@@ -6529,40 +6557,61 @@ private:
 
   * [2.3]{.pnum} `Accessor` meets the accessor policy requirements **[mdspan.accessor.reqmts]**.
 
+  * [2.4]{.pnum} _`scalable`_`<ScalingFactor, element_type>` is `true`. 
+
+  * [2.5]{.pnum} `is_constructible_v<element_type, decltype(declval<ScalingFactor>() * element_type)>` is `true`.
+
 ```c++
-constexpr accessor_scaled(const ScalingFactor& s, const Accessor& a);
+template<class OtherScalingFactor, class OtherNestedAccessor>
+  constexpr scaled_accessor(const scaled_accessor<OtherScalingFactor, OtherNestedAccessor>& other);
 ```
 
+[3]{.pnum} *Constraints:*
+
+  * [3.1]{.pnum} `is_constructible_v<`_`nested-accessor`_`, const OtherNestedAccessor&>` is `true`, and
+
+  * [3.2]{.pnum} `is_constructible_v<ScalingFactor, OtherScalingFactor>` is `true`.
+
 [3]{.pnum} *Effects:*
 
-  * [3.1]{.pnum} Direct non-list-initializes _`scaling-factor`_ with `s`, and
+  * [3.1]{.pnum} Direct non-list-initializes _`scaling-factor`_ with `other.scaling_factor()`, and
 
-  * [3.2]{.pnum} direct non-list-initializes _`nested-accessor`_ with `a`.
+  * [3.2]{.pnum} direct non-list-initializes _`nested-accessor`_ with `other.nested_accessor()`.
+
+```c++
+constexpr scaled_accessor(const ScalingFactor& s, const Accessor& a);
+```
+
+[4]{.pnum} *Effects:*
+
+  * [4.1]{.pnum} Direct non-list-initializes _`scaling-factor`_ with `s`, and
+
+  * [4.2]{.pnum} direct non-list-initializes _`nested-accessor`_ with `a`.
 
 ```c++
 constexpr reference access(data_handle_type p, size_t i) const noexcept;
 ```
 
-[4]{.pnum} *Returns:* `reference(`_`scaling-factor`_`,` _`nested-accessor`_`.access(p, i))`
+[5]{.pnum} *Returns:* `scaling_factor() * element_type(` _`nested-accessor`_`.access(p, i))`
```

Comment:
This looks right.

Reply:
Actually, perhaps not -- I think the cast needs to use the nested accessor's element type.

```suggestion
[5]{.pnum} *Returns:* `scaling_factor() * typename Accessor::element_type(` _`nested-accessor`_`.access(p, i))`
```